src/cript/api/schema_validation.py

```diff
@@ -0,0 +1,64 @@
+import json
+
+import requests
+from jsonschema import validate
+
+from cript.api.api import _get_global_cached_api
+
+_DB_SCHEMA: json = None
```

Comment:
This should be `_DB_SCHEMA: dict` right now it is a module type json.

Actual json data is dicts in python.

Reply:
done